docs/tooling/cli-guides/setup-a-devnet.md

```diff
@@ -758,3 +758,144 @@ curl http://52.204.202.216:9650/ext/bc/JjDfmxM3hAEX3VuaKH4PpQskhrvp2pzGTgYLpDwin
 
 C-Chain endpoint follows the same scheme but with the blockchain alias of `C`: `http://52.204.202.216:9650/ext/bc/C/rpc`
 
+<!-- vale off -->
+
+#### Deploy Teleporter
+
+<!-- vale on -->
+
+<!-- vale off -->
+
+[Teleporter](https://github.com/ava-labs/teleporter) is a cross-chain messaging protocol built on top of Warp. Teleporter provides a user-friendly interface to Warp that provides additional features such as replay protection, message delivery incentives, and message execution, to name a few.
+
+Now that the Warp-enabled devnet is up and running, Teleporter can be deployed to the C-Chain and subnet-evm subnets. Teleporter is deployed using [Nick's method](https://yamenmerhi.medium.com/nicks-method-ethereum-keyless-execution-168a6659479c) to ensure that the Teleporter contract is deployed to the same address on all chains. For instructions on how to deploy Teleporter, please see the guide [here.](https://github.com/ava-labs/teleporter/blob/main/utils/contract-deployment/README.md) Follow those instructions to deploy Teleporter to Subnet A, Subnet B, and the C-Chain.
+
+<!-- vale on -->
+
+#### Run an AWM Relayer instance
+
+<!-- vale off -->
+
+In order to deliver Warp and Teleporter messages between chains, an off-chain relayer is required. For this tutorial, we will use [AWM Relayer.](https://github.com/ava-labs/awm-relayer) 
+
+<!-- vale on -->
+
+> **_NOTE:_** In this tutorial, we'll run the relayer on the same local machine from which we deployed the devnets, but these steps can be adapted to deploy the relayer in a cloud environment as well. The application itself has minimal hardware requirements, and is essentially stateless.
+
+<!-- vale off -->
+
+##### Configure the Relayer
+
+<!-- vale on -->
+
+First, we need to set the configuration using our devnet values. Below is a template relayer configuration with the values for Subnet A (which we deployed previously) filled in:
```

Comment:
values are not filled in

Reply:
In the section below I describe the missing variables and how to populate them.